svalinn_tools/data_loading.py

```diff
@@ -135,7 +135,7 @@ def plane_slice_plotting(window_number, axis_number, label, images, timestamp):
 
     # Include the CNERG logo in the bottom left corner of the plot.
     image = Vi.CreateAnnotationObject("Image")
-    image.image = os.path.dirname(os.path.abspath(__file__)) + "/cnerg.jpg"
+    image.image = path.abspath(path.dirname(__file__)) + "/../img/cnerg.png"
```

Comment:
Maybe we should use the Svalinn logo rather than CNERG, since this is in the Svalinn org.

Reply:
Hmmm....  I guess the question is what kind of branding we want here.  As an open source Svalinn projects, I suppose it's possible for the authorship and brand to grow beyond CNERG, so it does make sense to have broader brand.  Let's be inclusive and go with the Svalinn logo.